Managed Services/persisting-wordpress-files.md

```diff
@@ -0,0 +1,88 @@
+{{{
+  "title": "Persisting Wordpress Files",
+  "date": "04-02-2015",
+  "author": "Bill Burge",
+  "attachments": [],
+  "contentIsHTML": true
```

Comment:
This should be set to true only if the entire document uses html syntax. Since this doc uses both markdown and html syntax – which is also valid markdown – it should be `false`. This is important for the tool that imports this content for use on centurylinkcloud.com. I'll add some info in the readme about this.

p.s. Generally, we'd like to see **new** kb articles written entirely in markdown syntax.